src/test/resources/server.yml

```diff
@@ -5,7 +5,7 @@ server:
     - type: http
       port: 8000
 
-webLoggerEnabled: false
+webLoggerEnabled: true
```

Comment:
enables weblogger so the instructions in readme.md for com.palantir.weblogger.examples.Example work